src/v/cluster/metadata_dissemination_service.cc

```diff
@@ -252,13 +252,19 @@ ss::future<> metadata_dissemination_service::process_get_update_reply(
         return ss::make_ready_future<>();
     }
     // Update all NTP leaders
-    return _leaders
-      .invoke_on_all([reply = std::move(reply_result.value())](
-                       partition_leaders_table& leaders) mutable {
-          for (auto& l : reply.leaders) {
-              leaders.update_partition_leader(l.ntp, l.term, l.leader_id);
-          }
-      })
+    vlog(clusterlog.trace, "updating leadership from get_metadata");
```

Comment:
Did you intend to grab a line from https://github.com/redpanda-data/redpanda/commit/a79c0450387bfb315761b57adeaebba24250dfe6?

Reply:
yeah that was a mistake when I was fixing conflicts in the cherry pick